packages/ember-metal/tests/mixin/merged_properties_test.js

```diff
@@ -91,6 +91,28 @@ test('mergedProperties should be concatenated', function() {
   deepEqual(get(obj, 'bar'), { a: true, l: true, e: true, x: true }, "get bar");
 });
 
+test("mergedProperties should exist even if not explicitly set on create", function(){
+
+  var AnObj = Ember.Object.extend({
+    mergedProperties: ['options'],
+    options: {
+      a: 'a',
+      b: {
+        c: 'ccc',
```

Comment:
Latest JSCS failure is for this trailing comma (which makes old IE not parse properly).